code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -31,6 +29,8 @@
 
 	create_internal_organs()
 
+	grant_language(/datum/language/monkey)

```

Comment:
probably worth some kind of ```var/list/init_languages``` that are given by default to the mob (I prefer things in definitions to constructors where possible)